data/mods/classic_zombies/design-doc.md

```diff
@@ -0,0 +1,35 @@
+
+Four months ago, new reports of the same effect came up almost simultaneously in Morocco, Belarus, Vietnam, and Portugal. The WHO expressed concern and gave advice for avoiding wild animals. Within a month, the plague had spread to most of the planet. The initial caution about wild animals proved a dangerous red herring, causing people to cluster in urban centers, where the actual vector - other humans and some infected vermin - spread the zombie affliction rapidly. Unfortunately the long latency of the agent meant that it was another month before the ubiquity of the infection became apparent. By the time the second wave of infections became clear, millions of people had been infected, and already passed it to the currently unaware third wave. Hospitals were rapidly overwhelmed, before the revivification risk was even clear. Critically, police and military were among those affected by the unrecognized first wave, and their ranks were as decimated as civilians.
+
+The first wave of zombies further spread the illness due to unreadiness and the population already being overwhelmed by the new pandemic. The death toll was catastrophic, particularly among medical personnel caring for those infected with the illness. It isn't known exactly what went wrong with military efforts, but they were undoubtedly stymied by their own ranks falling ill will the airborne form of the pandemic even as they tried to fight the zombies.
+
+Within three months, the world was collapsing. Survivors banded together in small groups, now aware of the signs and symptoms of the illness and the danger of a bite. You are one of those survivors. For the last few weeks you've been somewhere safe, holed up, waiting for things to die down, but it's clear now that things aren't going to die down.
+
+### Zombification Mechanics
+The zombie affliction is infectious, possibly viral or fungal. Survivors may never find out. It comes in two forms: it transmits by airborn spores, which have a low infection rate and a long latency. After infection, carriers are very quickly able to infect others, but do not develop symptoms for 2-3 weeks. Once symptomatic it progresses quickly, with death usually occuring in another week. The infectious symptoms are vague and flu-like: fever, chills, severe nausea and vomiting, muscle aches, muscle spasms, and eventually delirium and death. Due to the relatively low infection rate of the airborn form, only around half of the population were susceptible to this form, and all survivors are clearly among that group. However, the much higher dose of infectious agent shared by bite or even scratch from a reanimated zombie carries an increased risk of infection and generates a much faster progressing form of the illness.
```

Comment:
> Due to the relatively low infection rate of the airborn form, only around half of the population were susceptible to this form, and all survivors are clearly among that group.
The group that is susceptible? I am not sure this makes sense.
```suggestion
The zombie affliction is infectious, possibly viral or fungal. Survivors may never find out. It comes in two forms: it transmits by airborn spores, which have a low infection rate and a long latency. After infection, carriers are very quickly able to infect others, but do not develop symptoms for 2-3 weeks. Once symptomatic it progresses quickly, with death usually occurring in another week. The infectious symptoms are vague and flu-like: fever, chills, severe nausea and vomiting, muscle aches, muscle spasms, and eventually delirium and death. Due to the relatively low infection rate of the airborne form, only around half of the population were susceptible to this form, and all survivors are clearly among that group. However, the much higher dose of infectious agent shared by bite or even scratch from a reanimated zombie carries an increased risk of infection and generates a much faster progressing form of the illness.
```

Reply:
I would likewise think the survivors would be clearly NOT in the susceptible group.